### DISCUSSION TITLE
be节点一直去连本地127.0.0.1的fe节点，日志长期存在报错

### DISCUSSION THREAD

**Discussion by tilepotter** [apache/doris#26649] [RFC]
be节点一直去连本地127.0.0.1的fe，但实际上fe节点是在其他机器上，已经在fe节点通过这个命令添加了be节点：ALTER SYSTEM ADD BACKEND "be_host:heartbeat-service_port";
日志一直有报错如下：
I1109 15:23:56.808331 27112 load_channel_mgr.cpp:179] cleaning timed out load channels
I1109 15:23:56.808357 27112 load_channel_mgr.cpp:211] load mem consumption(bytes). limit: 13420326092, current: 0, peak: 0, total running load channels: 0
I1109 15:23:57.314558 27244 segment_loader.cpp:102] prune 0 entries in segment cache. cost(ms): 0
I1109 15:23:57.332173 27253 olap_server.cpp:714] cooldown producer get tablet num: 0
I1109 15:23:57.667536 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[3]...
W1109 15:23:57.667603 27338 utils.cpp:140] fail to get master client from cache. host=127.0.0.1, port=9020, code=7
I1109 15:24:00.509992 27334 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[1]...
I1109 15:24:02.510380 27334 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[2]...
I1109 15:24:03.668112 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[1]...
I1109 15:24:05.668512 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[2]...
I1109 15:24:06.510792 27334 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[3]...
W1109 15:24:06.510825 27334 utils.cpp:89] fail to get master client from cache. host=127.0.0.1, port=9020, code=[THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接)
W1109 15:24:06.510862 27334 task_worker_pool.cpp:1482] failed to report TASK|host=127.0.0.1|port=9020|error=[INTERNAL_ERROR]Fail to get master client from cache
I1109 15:24:09.668900 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[3]...
W1109 15:24:09.668941 27338 utils.cpp:140] fail to get master client from cache. host=127.0.0.1, port=9020, code=7
I1109 15:24:09.763163 27650 heartbeat_server.cpp:47] get heartbeat from FE.host:127.0.0.1, port:9020, cluster id:1019014950, counter:625, BE start time: 1699511457346
I1109 15:24:15.669507 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[1]...
I1109 15:24:17.332296 27253 olap_server.cpp:714] cooldown producer get tablet num: 0
I1109 15:24:17.669883 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[2]...
I1109 15:24:20.511461 27334 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[1]...
I1109 15:24:21.670266 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[3]...
W1109 15:24:21.670306 27338 utils.cpp:140] fail to get master client from cache. host=127.0.0.1, port=9020, code=7
I1109 15:24:22.511833 27334 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[2]...
I1109 15:24:26.512192 27334 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[3]...
W1109 15:24:26.512228 27334 utils.cpp:89] fail to get master client from cache. host=127.0.0.1, port=9020, code=[THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接)
W1109 15:24:26.512262 27334 task_worker_pool.cpp:1482] failed to report TASK|host=127.0.0.1|port=9020|error=[INTERNAL_ERROR]Fail to get master client from cache
I1109 15:24:27.670874 27338 client_cache.h:167] Failed to get client from cache: [THRIFT_RPC_ERROR]Couldn't open transport for 127.0.0.1:9020 (socket open() error: 拒绝连接), retrying[1]...

**Comment by LemonLiTree** [comment]


**Reply by tilepotter** [reply]
Thank you, I now delete the metadata directory of the FE node and the data storage directory of the BE node, and reconfigure the priority_networks of FE and BE, and redeploy the cluster to solve the problem.

**Reply by LemonLiTree** [reply]
ok
